internal/telemetry/traces.go

```diff
@@ -17,9 +17,15 @@ import (
 
 // TracerProvider provides Tracers that are used by instrumentation code to
 // trace computational workflows.
-func TracerProvider(ctx context.Context, logger logr.Logger, cfg Config, appName, appVersion string) (trace.TracerProvider, ShutdownProvider, error) {
+func TracerProvider(
+	ctx context.Context,
+	logger logr.Logger,
+	cfg Config,
+	appName, appVersion string,
+) (trace.TracerProvider, ShutdownProvider, error) {
 	if !cfg.Traces.Enabled || cfg.Traces.Address == "" {
-		logger.V(1).Info("Tracing system is disabled.", "enabled", cfg.Traces.Enabled, "addr", cfg.Traces.Address, "sampling-ration", cfg.Traces.SamplingRatio)
+		logger.V(1).
+			Info("Tracing system is disabled.", "enabled", cfg.Traces.Enabled, "addr", cfg.Traces.Address, "sampling-ration", cfg.Traces.SamplingRatio)
```

Comment:
This isn't great, but the alternative is to use `golines --nochain-split-dots` which will break this simple logging entry into something like the following:

```go
logger.V(1).
	Info(
		"Starting new workflow",
		"watcher",
		event.WatcherName,
		"bucket",
		event.Bucket,
		"key",
		event.Key,
		"dir",
		event.IsDir,
	)
```

I'd rather fix this specific case manually.

Reply:
Hmm, in general I like the `--no-chain-split-dots` style better then the default (See: https://github.com/segmentio/golines?tab=readme-ov-file#chained-method-splitting), but I agree that it's bad with these key / value pairs.